src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -162,12 +162,14 @@ TypeAnnotations BuildTypeAnnotations (TypeDefinition type)
 								paramAnnotations[0] = methodMemberTypes;
 							}
 						} else if (methodMemberTypes != DynamicallyAccessedMemberTypes.None) {
-							_context.LogWarning ($"The DynamicallyAccessedMembersAttribute is only allowed on method parameters, return value or generic parameters.", 2041, method);
+							_context.LogWarning ($"The DynamicallyAccessedMembersAttribute is only allowed on method parameters, return value or generic parameters.",
```

Comment:
Do we need to worry about localization?

Reply:
We currently don't localize any messages outputted by the linker. I sincerely don't know how useful that would be, but might be a good next step after polishing the current messages that we have (https://github.com/mono/linker/issues/1275).

Reply:
Not for .NET 5 - linker is not localized (at all). If we think it's important it would be a relatively non-trivial feature on its own (mostly because there's nothing in that space in linker, so we would have to introduce everything from scratch). So .NET 6 (if we think it's important).

Reply:
Are other parts of the tool chain localized? What happens for Csc.exe?

Reply:
Yeah, csc.exe is fully localized. Visual Studio requires localization in all core languages as a shipping criteria, so it was a hard requirement for us. That doesn't sound like the case for the linker, so let's come back to that in the 6.0 timeframe as Vitek suggests.